src/index.ts

```diff
@@ -0,0 +1,9 @@
+import readingTime from './reading-time'
+import ReadingTimeStream from './stream'
+
+// Wacky way to support const readingTime = require('reading-time') :(
+// Basically we can't use ES import/export anymore because re-assigning module.exports
+// decouples it from the exports object, which TS export compiles to
+module.exports = readingTime
+module.exports.default = readingTime
+module.exports.ReadingTimeStream = ReadingTimeStream
```

Comment:
I think this should work:

```suggestion
import readingTimeFn from "./reading-time";
export default readingTimeFn;
export const readingTime = readingTimeFn;

export { default as ReadingTimeStream } from "./stream";

```

Reply:
The problem with `export default readingTimeFn;` is it's transpiled to `module.exports.default = readingTimeFn`. I need to support the `require()` without `.default` because that seems less straightforward.

I've asked the TS community if they have anything good and let's wait for a couple of days to see 👀